packages/suite/src/components/suite/NavigationBar/index.tsx

```diff
@@ -19,7 +19,7 @@ const StyledNavigationBar = styled.div<{ isMobileLayout: boolean }>`
     background: ${props => props.theme.BG_WHITE};
     border-bottom: 1px solid ${props => props.theme.STROKE_GREY};
 
-    &:hover ${StyledDeviceSelector} {
+    &${StyledDeviceSelector}:hover {
```

Comment:
How does this work?

Reply:
Current code triggers hover styles on `StyledDeviceSelector` component when its parent is hovered. I changed it to trigger hover styles when `StyledDeviceSelector` is hovered, not its parent